src/main/java/org/eclipse/foundation/marketplace/dto/Category.java

```diff
@@ -69,57 +65,49 @@ public void setUrl(String url) {
 		this.url = url;
 	}
 
-	/**
-	 * @return the marketIds
-	 */
-	public List<Integer> getMarketIds() {
-		return new ArrayList<>(marketIds);
-	}
-
-	/**
-	 * @param marketIds the marketIds to set
-	 */
-	public void setMarketIds(List<Integer> marketIds) {
-		this.marketIds = new ArrayList<>(marketIds);
-	}
-
 	@Override
 	public int hashCode() {
```

Comment:
Don't code hashcode by yourself, use Object.hash(Object...) https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#hash-java.lang.Object...-

Reply:
Same as the object.equals stuff, auto-generated. I'll switch over.